app/views/chess_boards/index.html.erb

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>ChessBoard</title>
+
+
+
+  </head>
+  <body>
+    <br><br><br>
+
+
+    <div class="chessboard">
+<!-- 1st -->
+<div class="white white-top white-left"></div>
```

Comment:
Lots of repeated code here. General rule of thumb - if you are ever copy/pasting, you are doing it wrong.

In this instance, you can loop 0-8, with an inner 0-8 loop. You can then identify black/white by using modulo. Hint on the modulos is `4 % 2 == 0 && 5 % 2 == 1`